README.md

```diff
@@ -244,13 +244,11 @@ application, you should __always__ benchmark if performance matters to you.
 * [__`Serverless`__](./docs/Guides/Serverless.md)
 * [__`Recommendations`__](./docs/Guides/Recommendations.md)
 
-中文文档[地址](https://github.com/fastify/docs-chinese/blob/HEAD/README.md)
```

Comment:
Got rid of the link to the Chinese docs as that is very out of date and should probably be archived alongside the other language-variant documentation repos.